source/views/streaming/streams/list.js

```diff
@@ -92,6 +96,12 @@ export class StreamListView extends React.PureComponent {
       const grouped = groupBy(processed, j => j.$groupBy)
       const mapped = toPairs(grouped).map(([title, data]) => ({title, data}))
 
+      // wait 0.5 seconds – if we let it go at normal speed, it feels broken.
+      const elapsed = start - Date.now()
```

Comment:
This is the incorrect code that I noticed over the weekend… it should be `Date.now() - start`.

Reply:
Ah... thanks.